RecoTracker/LSTCore/src/alpaka/Event.dev.cc

```diff
@@ -1,7 +1,8 @@
 #include "Event.h"
 
-void SDL::Event<SDL::Acc>::init(bool verbose) {
+void SDL::Event<SDL::Acc>::init(bool verbose, float pt_cut) {
   addObjects = verbose;
+  ptCut = pt_cut;
```

Comment:
My concern was to avoid issues with shadowing. I suppose I could change the argument to ptCut and change this line to:

this->ptCut = ptCut;

What do you think?

Reply:
Link to the mentioned code below.

https://github.com/SegmentLinking/cmssw/blob/54304f0ee7235aa45660853fec0f575cd7a2c8ca/RecoTracker/LSTCore/src/alpaka/Event.h#L100-L113

Reply:
If it causes warning, then it is fine for me to stay as is. It was more of a comment for our convenience (not creating too many symbols).